sample-config-files/defender-config.yaml

```diff
@@ -46,4 +46,8 @@ Defender:
   MS.DEFENDER.2.1v1 : &UserImpersonationProtection
     SensitiveAccounts:
       IncludedUsers:
-        - "John Doe;jdoe@someemail.com"
\ No newline at end of file
+        - "John Doe;jdoe@someemail.com"
+  MS.DEFENDER.2.3v1 : &DomainImpersonationProtection
+    SensitiveAccounts:
+      IncludedDomains:
+        - "random.mail.example.com"
```

Comment:
Nitpick. Just wondering about consistency. Some of these are wrapped in quotes, e.g., `"random.mail.example.com"`, but not `johndoe@random.example.com`. Both styles are valid. But maybe we'd want to have all with quotes or all without for consistency.

Reply:
Copy and paste error, remove the string quotes